src/components/explorer/suggestion/SuggestionList.tsx

```diff
@@ -29,7 +29,6 @@ export const SuggestionList: FC<Props> = (props) => {
             className="px-30 flex cursor-pointer items-center space-x-10 py-10  hover:bg-white/20 aria-selected:bg-white/10"
             to="/explore/$type/$slug"
             params={params}
-            search={{}}
```

Comment:
Nice, the empty objects on `search` and `params` can now be removed from these files too:
- src/components/core/menu/mainMenu/MainMenuLeft.tsx
- src/components/core/menu/mobileMenu/MobileMenu.tsx
- src/components/strategies/StrategyPageTabs.tsx
- src/components/trade/TradeExplorerTabs.tsx